threema.gemspec

```diff
@@ -16,21 +16,19 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/thorsteneckel/threema'
   spec.license       = 'MIT'
 
-  spec.required_ruby_version = '>= 2.6.0'
+  spec.required_ruby_version = '>= 2.7.0'
 
   spec.files         = Dir['{lib}/**/*']
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'case_transform'
+  spec.add_runtime_dependency 'activesupport', '~> 7.0.8'
```

Comment:
It would be nice to only import active support's string inflections. But I'm not aware how this is possible in Ruby land.